server/server.py

```diff
@@ -327,6 +334,21 @@ def do_GET(self):
         contenttype = 'application/json'
         if self.path == '/devices':
             message = json.dumps(deviceslist)
+        elif self.path == '/help':
```

Comment:
This is a good addition.

Reply:
I debated if it would be better as an HTML payload, but the JSON response still provides the help.